<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewJobs</class>
 <widget class="QMainWindow" name="ViewJobs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>589</width>
    <height>252</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vis innlagte jobber</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>jobber.png</normaloff>jobber.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelSoknadID">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>64</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Søknadid*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxApplicationID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>61</y>
      <width>451</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTittel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>96</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Tittel*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTitle">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>93</y>
      <width>451</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelBedrift">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>128</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Bedrift*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCompany">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>125</y>
      <width>451</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelStedID">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>159</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Stedsnavn*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTownID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>156</y>
      <width>451</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatusID">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>191</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Status*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStatusID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>188</y>
      <width>451</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSoknadsfrist">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>223</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Søknadsfrist*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDeadline">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>451</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFirst">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>420</y>
      <width>82</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrev">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>420</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>420</y>
      <width>82</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLast">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>420</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Lagre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Slett</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>9</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Her kan du se informasjon om søknadene som er registrert i databasen. Felt markert med * må fylles ut</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMotivation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Motivasjon</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMotivation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>451</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxApplicationID</tabstop>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>lineEditCompany</tabstop>
  <tabstop>comboBoxTownID</tabstop>
  <tabstop>comboBoxStatusID</tabstop>
  <tabstop>lineEditDeadline</tabstop>
  <tabstop>textEditMotivation</tabstop>
  <tabstop>btnFirst</tabstop>
  <tabstop>btnPrev</tabstop>
  <tabstop>btnNext</tabstop>
  <tabstop>btnLast</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
